### DISCUSSION TITLE
Widget Windows 11 - AllowMultiple="true" Doesn't work?

### DISCUSSION THREAD

**Discussion by EdwinVD** [microsoft/WindowsAppSDK#3644] [Q&A]
Hello there,

I'm currently testing a widget using your WindowAppSDK. I've used the sample code provided on your repository, and I've tried to set "AllowMultiple=true" in order to have multiple instances of the same widget. However, it doesn't seem to be working as expected.
Source XML : https://learn.microsoft.com/en-us/windows/apps/develop/widgets/widget-provider-manifest

I wanted to reach out and inquire if this is still a known issue or if there might be any possible solutions. I'm currently using Windows 11 with build 25336.1000.

Let me know how to resolve this issue or what build this is fixed.

Thanks!

**Comment by beervoley** [comment]
Hi @EdwinVD, on which verison of Widgets Dashboard do you observe this behavior? To check the version:
Click on your avatar in the right top corner of the dashboard:
![image](https://github.com/microsoft/WindowsAppSDK/assets/21356912/21ae4efb-24d5-4bba-ba18-758681c71d62)

**Reply by EdwinVD** [reply]
Hey @beervoley 
I'm using this version. 523.23300.0. (Windows 11 22H 235411000) DEV
<img width="646" alt="Screenshot 2023-09-12 at 10 00 12 PM" src="https://github.com/microsoft/WindowsAppSDK/assets/16556638/b3571e6b-d48c-4984-9063-5d629e69bf99">

So all the widgets got this parameter added. if you want to test see MS Store widgets.

**Comment by btigi** [comment]
I'm still seeing this issue, whereby I have added AllowMultiple to the definition but I'm unable to add more than one instance of the widget to the widget dashboard.

Widget Dashboard: 524.34401.20.0
Microsoft.WindowsAppSDK: 1.6.250108002
Windows 11 23H2
